### PR TITLE
adding tree view.

### PR DESCRIPTION
**Pull request by danyaljj**
screen shot 2017-04-02 at 10 54 40 am

### CODE REVIEW

**Comment by danr-ccg** [comment]
Hi Daniel,

Can you explain what that is? Can you run it also on Illinois dependencies?

              Dan

**Comment by danyaljj** [comment]
This is a generic visualization and it works for anything we have in the pipeline. 
So the answer is: yes! We can use it for our own parser.

File: src/renderers/treeViewRenderer.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/* eslint no-console:0 */
+
+var _ = require('lodash');
+var randomColor = require('randomcolor');
+
+const supportedTypes = ["edu.illinois.cs.cogcomp.core.datastructures.textannotation.TreeLabelView"];
```
Comment (bhargav) [inline src/renderers/treeViewRenderer.js]: This should be `edu.illinois.cs.cogcomp.core.datastructures.textannotation.TreeView`

Comment (danyaljj) [inline src/renderers/treeViewRenderer.js]: fixed.

**Pull request merged**